### PR TITLE
Added multiple postgres instances and fixed pgbouncer

### PR DESCRIPTION
**Pull request by rothgar**
This fixes a problem with pgbouncer class not including the datadog_agent (error if not otherwise included)

It also allows multiple postgres instances on the same node. It's based on the mysql template and manifest but now can be applied to postgres.

### CODE REVIEW

**Comment by rothgar** [comment]
Just for reference/testing, here's an example to declare multiple postgres instances in hiera for a node

```
datadog_agent::integrations::postgres::instances:
  - host: foo
    port: 5432
    username: postgres
    password: foo_pass
    dbname:
    ssl: False
    tags:
      - Foo DB
      - postgresql
  - host: bar
    port: 5400
    username: postgres
    password: bar_pass
    ssl: False
    tags:
      - Bar DB
      - postgresql
```

File: manifests/integrations/postgres.pp [hunk 1]
```diff
@@ -74,7 +75,25 @@
...
+      'tables'         => $tables,
+      'custom_metrics' => $custom_metrics,
+    }]
+  } elsif !$instances{
+    $_instances = []
+  } else {
+    $_instances = $instances
+  }
+
+  file { "$dst":
```
Comment (truthbk) [inline manifests/integrations/postgres.pp]: This line has a minor stylistic issue:
```
manifests/integrations/postgres.pp:96:only_variable_string:WARNING:string containing only a variable
manifests/integrations/postgres.pp:96:variables_not_enclosed:WARNING:variable not enclosed in {}
```

The line was originally:
```
  file { $dst:
```
which I think is fine, but if you have your reasons for changing that you might have to address it some other way.

**Comment by rothgar** [comment]
I reverted the file definition. Do you know of an example for testing passing an array of instances? I looked in the mysql tests but didn't see a test for an array so I'm not sure how to do it.

**Comment by truthbk** [comment]
@rothgar that's OK, I'm working on puppet right now, so I'll add the tests to the suite myself. Thank you very much for your work.

**Pull request merged**